bug_reports/Bash_JSONPath.sh.md

```diff
@@ -46,7 +46,7 @@ The following queries provide results that do not match those of other implement
   ```
   Error:
   ```
-  ggrep: Invalid range end
+  grep: Invalid range end
```

Comment:
Just FYI, this is a difference between Linux and OS X (using homebrew). For now I'm personally happy to ignore the noise.